modules/system/ServiceProvider.php

```diff
@@ -94,6 +94,7 @@ public function boot()
             }
         }
 
+        Paginator::useBootstrapThree();
         Paginator::defaultSimpleView('system::pagination.simple-default');
```

Comment:
@FlusherDock1 you can feel free to override this in your own plugins but changing this away from Bootstrap 3 will break existing sites using the paginate()->render().

Reply:
@LukeTowers I respect that, but using this kind of logic we can't updated everything because there will be always one website that can be broken because of the new changes. 

P.S. Bootstrap 4 was released 6 years ago, I don't think someone really using Bootstrap 3 codebase right now.

Reply:
@FlusherDock1 there's no-one stopping you from using whatever markup you want in your own projects. I can tell you right now that there are active sites with millions of views annually that utilize the default Bootstrap 3 markup. If you want to use Bootstrap 4, then just do the following in your own plugins:

```php
Paginator::defaultView('pagination::bootstrap-4');
Paginator::defaultSimpleView('pagination::simple-bootstrap-4');
```

Reply:
@FlusherDock1 The `useBootstrapThree` call is mainly for the Backend, which uses the Storm UI and thus uses Bootstrap 3 classes. Changing this would break the pagination on the backend. As @LukeTowers has indicated, you are more than free to use Bootstrap 4 for your own plugins.

Reply:
>If you want to use Bootstrap 4, then just do the following in your own plugins:
Paginator::defaultView('pagination::bootstrap-4');
Paginator::defaultSimpleView('pagination::simple-bootstrap-4');

Thanks, i didn't know about that.

>The useBootstrapThree call is mainly for the Backend, which uses the Storm UI and thus uses Bootstrap 3 classes. Changing this would break the pagination on the backend. As @LukeTowers has indicated, you are more than free to use Bootstrap 4 for your own plugins.

About backend bootstrap 3 I know, I just suggested changes only for frontend